cluster/terraform_kubernetes/config/scripts/hash_password.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Can we move this script to the /scripts folder and call it manually instead of calling it via terraform?

Reply:
fixed this